frontend/src/app/core/components/sign-up/sign-up.component.ts

```diff
@@ -1,6 +1,10 @@
 import { Component, OnInit } from '@angular/core';
 import { AuthenticationService } from '@core/services/authentication.service';
 import { Router } from '@angular/router';
+import { AngularFireAuth } from '@angular/fire/auth';
+import { UserService } from '@core/services/user.service';
+import { RegisterDialogService } from '@core/services/register-dialog.service';
+import * as firebase from 'firebase';
```

Comment:
Import only needed element, not a whole firebase library. Fix it in other places too